commit cc2e0d17fbc6cee7f19042cf2113179659887802
Author: Bjorn Helgaas <bhelgaas@google.com>
Date:   Fri Mar 17 00:48:24 2017 +0000

    PCI: Update BARs using property bits appropriate for type
    
    [ Upstream commit 45d004f4afefdd8d79916ee6d97a9ecd94bb1ffe ]
    
    The BAR property bits (0-3 for memory BARs, 0-1 for I/O BARs) are supposed
    to be read-only, but we do save them in res->flags and include them when
    updating the BAR.
    
    Mask the I/O property bits with ~PCI_BASE_ADDRESS_IO_MASK (0x3) instead of
    PCI_REGION_FLAG_MASK (0xf) to make it obvious that we can't corrupt bits
    2-3 of I/O addresses.
    
    Use PCI_ROM_ADDRESS_MASK for ROM BARs.  This means we'll only check the top
    21 bits (instead of the 28 bits we used to check) of a ROM BAR to see if
    the update was successful.
    
    Signed-off-by: Bjorn Helgaas <bhelgaas@google.com>
    Signed-off-by: Sasha Levin <alexander.levin@verizon.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1428_cc2e0d17fbc6cee7f19042cf2113179659887802.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1428_cc2e0d17fbc6cee7f19042cf2113179659887802.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
